### PR TITLE
Change 'node selector` to 'label' and fix secret name in example

### PR DESCRIPTION
**Pull request by mburke5678**
Change _node selector_ to _label_: https://github.com/openshift/openshift-docs/issues/29478

Preview: https://deploy-preview-30221--osdocs.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-node-selectors.html#nodes-scheduler-node-selectors-pod_nodes-scheduler-node-selectors

https://deploy-preview-30221--osdocs.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-node-selectors.html#nodes-scheduler-node-selectors-project_nodes-scheduler-node-selectors

Change name of secret (and output): https://github.com/openshift/openshift-docs/issues/29255

Preview: https://deploy-preview-30221--osdocs.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-secrets.html#nodes-pods-secrets-certificates-creating_nodes-pods-secrets

### CODE REVIEW

**Comment by mburke5678** [comment]
@lyman9966 Is this an issue you can review? Thank you.

File: modules/nodes-pods-secrets-certificates-creating.adoc [hunk 1]
```diff
@@ -56,36 +56,34 @@ $ oc get secrets
 .Example output
 [source,terminal]
 ----
-NAME                         TYPE                                  DATA      AGE
-my-cert                  kubernetes.io/tls                     2         9m
+NAME                           TYPE                                  DATA      AGE
+my-service-pod                 kubernetes.io/tls                     2         9m
```
Comment (lyman9966) [inline modules/nodes-pods-secrets-certificates-creating.adoc:60]: this should not update, for we keep the secret name "my-cert" althrough the article

File: modules/nodes-scheduler-node-selectors-cluster.adoc [hunk 2]
```diff
@@ -137,13 +137,13 @@ After making this change, wait for the pods in the `openshift-kube-apiserver` pr
 
 * Use a `MachineSet` object to add labels to nodes managed by the machine set when a node is created:
 
-.. Run the following command to add a node selector to a `MachineSet` object:
+.. Run the following command to add a label to a `MachineSet` object:
```
Comment (lyman9966) [inline modules/nodes-scheduler-node-selectors-cluster.adoc:140]: I thinkd it's better to replace "a label" with "labels", for there are two pairs of <key, value>

File: modules/nodes-scheduler-node-selectors-pod.adoc [hunk 3]
```diff
@@ -64,7 +64,7 @@ The web console lists the controlling object under `ownerReferences` in the pod
 
 * Use a `MachineSet` object to add labels to nodes managed by the machine set when a node is created:
 
-.. Run the following command to add a node selector to a `MachineSet` object:
+.. Run the following command to add a label to a `MachineSet` object:
```
Comment (lyman9966) [inline modules/nodes-scheduler-node-selectors-pod.adoc:67]: I think it's better to replace "a label" with "labels", for there are two pairs of <key, value>

**Comment by lyman9966** [comment]
@mburke5678  I think you also need to update the node selector to labels in file: 
https://deploy-preview-30221--osdocs.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-node-selectors.html#nodes-scheduler-node-selectors-cluster_nodes-scheduler-node-selectors

**Comment by mburke5678** [comment]
@lyman9966 I think I got all the references to labels and node selectors correct. Thank you for your help.

File: modules/nodes-pods-secrets-certificates-creating.adoc (hunk 1 above)
Comment (lyman9966) [reply modules/nodes-pods-secrets-certificates-creating.adoc:60]: Hi, @mburke5678 , I think you should revert to "my-cert", not "my-service-pod"

Comment (lyman9966) [reply modules/nodes-pods-secrets-certificates-creating.adoc:60]: others look good to me

Comment (mburke5678) [reply modules/nodes-pods-secrets-certificates-creating.adoc:60]: @lyman9966 I apologize. I thought I changed that. I must have missed it. Thank you!

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Pull request merged**